### PR TITLE
Log operations to syslog

### PR DESCRIPTION
**Pull request by zmc**
To be honest I'm not entirely sure if this will be 100% safe in all setups.

That said, we've been using this branch as the main downburst copy on ``teuthology.front`` for weeks, and that is responsible for (guessing) 99.9% of all downburst usage.

### CODE REVIEW

File: downburst/create.py [hunk 1]
```diff
@@ -205,6 +207,23 @@ def create(args):
...
     dom.create()
+    try:
+        env = os.environ
+        pid = os.getpid()
+        # os.getppid() wont return the correct value:
+        ppid = open('/proc/{pid}/stat'.format(pid=pid)).read().split()[3]
+        ppcmdline = open('/proc/{ppid}/cmdline'.format(ppid=ppid)).read().split(b'\x00')
+
+    except IndexError, IOError:
+        log.error('Something went wrong getting PPID/cmdlineinfo')
```
Comment (alfredodeza) [inline downburst/create.py]: `log.error` will not show the exception and the exception is not being used to log what went wrong. Consider adding (or using) `log.exception` and/or using the exception object to log what went wrong

File: downburst/destroy.py [hunk 2]
```diff
@@ -55,6 +57,23 @@ def destroy(args):
...
         log.debug('Terminating the virtual machine')
+        env = os.environ
+        try:
+            pid = os.getpid()
+            # os.getppid() wont return the correct value:
+            ppid = open('/proc/{pid}/stat'.format(pid=pid)).read().split()[3]
+            ppcmdline = open('/proc/{ppid}/cmdline'.format(ppid=ppid)).read().split(b'\x00')
+
+        except IndexError, IOError:
+            log.error('Something went wrong getting PPID/cmdlineinfo')
```
Comment (andrewschoen) [inline downburst/destroy.py]: log.exception here as well, please

**Comment by andrewschoen** [comment]
If this was the branch being used in sepia then I imagine we're probably safe, at least for our purposes.

**Comment by zmc** [comment]
Good points on ``log.exception`` vs. ``log.error``; I'll update this soonish

**Pull request merged**